packages/strapi-admin/admin/src/containers/Onboarding/index.js

```diff
@@ -1,217 +1,136 @@
-          )}
-        >
-          <div className="videosHeader">
-            <p>
-              <FormattedMessage id="app.components.Onboarding.title" />
-            </p>
-            {videos.length && (
-              <p>
-                {Math.floor(
-                  (videos.filter(v => v.end).length * 100) / videos.length
-                )}
-                <FormattedMessage id="app.components.Onboarding.label.completed" />
-              </p>
+  const hasVideos = videos.length > 0;
+  const className = hasVideos ? 'visible' : 'hidden';
+
+  return (
+    <Wrapper className={className} isOpen={isOpen}>
+      <div className={cn('videosContent', isOpen ? 'shown' : 'hide')}>
+        <div className="videosHeader">
+          <p>
+            <FormattedMessage id="app.components.Onboarding.title" />
+          </p>
+          <p>
+            {Math.floor(
+              (videos.filter(v => v.end).length * 100) / videos.length
             )}
-          </div>
-          <ul className="onboardingList">
-            {videos.map((video, i) => {
-              return (
-                <OnboardingVideo
-                  key={i}
-                  id={i}
-                  video={video}
-                  onClick={onClick}
-                  setVideoDuration={setVideoDuration}
-                  getVideoCurrentTime={this.updateCurrentTime}
-                  didPlayVideo={this.didPlayVideo}
-                  didStopVideo={this.didStopVideo}
-                />
-              );
-            })}
-          </ul>
-        </div>
-
-        <div className="openBtn">
-          <button
-            onClick={this.handleVideosToggle}
-            className={this.state.showVideos ? 'active' : ''}
-            type="button"
-          >
-            <i className="fa fa-question" />
-            <i className="fa fa-times" />
-            <span />
-          </button>
+            <FormattedMessage id="app.components.Onboarding.label.completed" />
+          </p>
         </div>
-      </Wrapper>
-    );
-  }
-}
-
-Onboarding.defaultProps = {
-  onClick: () => {},
-  removeVideos: () => {},
-  setVideoDuration: () => {},
-  setVideoEnd: () => {},
-  shouldOpenModal: false,
-  videos: [],
-  updateVideoStartTime: () => {},
-};
-
-Onboarding.propTypes = {
-  getVideos: PropTypes.func.isRequired,
-  onClick: PropTypes.func,
-  removeVideos: PropTypes.func,
-  setVideoDuration: PropTypes.func,
-  setVideoEnd: PropTypes.func,
-  shouldOpenModal: PropTypes.bool,
-  updateVideoStartTime: PropTypes.func,
-  videos: PropTypes.array,
-};
-
-const mapStateToProps = makeSelectOnboarding();
-
-function mapDispatchToProps(dispatch) {
-  return bindActionCreators(
-    {
-      getVideos,
-      onClick,
-      setVideoDuration,
-      updateVideoStartTime,
-      setVideoEnd,
-      removeVideos,
-    },
-    dispatch
+        <ul className="onboardingList">
+          {videos.map((video, index) => {
+            return (
```

Comment:
Useless return statement here.

Reply:
In fact I find it useful to add the return statement especially when we need to debug or log what is happening... You'll see that I often leave the return statement for this particular reason.

Reply:
You can add a log without adding a return statement. As the console.log fonction return always false, you just need to do this : 
```jsx
{videos.map((video, index) => console.log('log') || (
     <Video ... />
)}
```